packages/docs-page/server/loaders/remote-content.ts

```diff
@@ -186,9 +199,10 @@ export default class RemoteContentLoader implements DataLoader {
     if (document.githubFile) {
       // Link latest version to `main`
       // Hide link on older versions
-      const isLatest = versionMetadataList.find(
-        (e) => e.version === document.version
-      )!.isLatest
+      const isLatest =
+        (versionFromPath === 'latest' && Boolean(this.opts.latestVersionRef)) ||
+        versionMetadataList.find((e) => e.version === document.version)!
```

Comment:
Is this non-null assertion safe to do (in that `versionMetadataList` will _always_ have a vertion that matches `document.version`)?

Reply:
Yeah, if we get to this point that should be the case.